language/de-DE/de-DE.ini

```
@@ -44,6 +45,7 @@ JACTION_EDIT="Bearbeiten"
 JACTION_EDITOWN="Eigene Inhalte bearbeiten"
 JACTION_EDITSTATE="Status bearbeiten"
 JACTION_LOGIN_ADMIN="Administratoranmeldung"
+JACTION_LOGIN_API="Webservice-Anmeldung"
```

Alternative 1:

```suggestion
JACTION_LOGIN_API="Web Service Anmeldung"
```

---

Alternative 2:

```suggestion
JACTION_LOGIN_API="Webserviceanmeldung"
```

zu lang!

---

mir gefällt der aktuelle String am besten